clojure-mode.el

```diff
@@ -524,6 +524,35 @@ replacement for `cljr-expand-let`."
                  #'clojure-space-for-delimiter-p)
     (advice-add 'paredit-convolute-sexp :after #'clojure--replace-let-bindings-and-indent)))
 
+(defun clojure-current-defun-name ()
+    "Return the name of the defun at point, or nil."
```

Comment:
The indentation of the entire function seems off (it's 2 spaces too indented), so you might want to use `indent-region` on it. 

I'd also suggest expanding the docstring to mention this function is meant to be used by `which-func`.